src/app/app.component.html

```diff
@@ -5,5 +5,6 @@ <h1>
 <app-tjs-multiplication></app-tjs-multiplication>
 <app-krl-color-mixer></app-krl-color-mixer>
 <app-pizza-gluer></app-pizza-gluer>
+<app-bs-list-generator></app-bs-list-generator>
```

Comment:
Can you move your component to the bottom?

Reply:
How do I do that?

Reply:
I think I got it :)